### PR TITLE
fix: copy of login screen with password and setting screen

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -414,6 +415,9 @@
   "dialogClearAllDataDeletionFailedTitleQuestion": "Do you want to delete data from just this device?",
   "dialogClearAllDataDeletionFailedMultiple": "Data not deleted by those Service Nodes: $snodes$",
   "dialogClearAllDataDeletionQuestion": "Would you like to clear this device only, or delete your data from the network as well?",
+  "clearDevice": "Clear Device",
+  "tryAgain": "Try Again",
+  "areYouSureClearDevice": "Are you sure you want to delete your device?",
```
Comment (yougotwill) [inline _locales/en/messages.json:420]: This should say clear device not delete device as in Figma.

File: ts/mains/password_start.tsx [hunk 2]
```diff
@@ -3,10 +3,13 @@
 
 import React from 'react';
 import { SessionPasswordPrompt } from '../components/SessionPasswordPrompt';
+import { SessionToastContainer } from '../components/SessionToastContainer';
 import { SessionTheme } from '../state/ducks/SessionTheme';
 
 window.ReactDOM.render(
   <SessionTheme>
+    <SessionToastContainer />
+
```
Comment (yougotwill) [inline ts/mains/password_start.tsx:12]: remove new line?

**Review by yougotwill — CHANGES_REQUESTED** [review]
Nice work. Minor copy fixes.

File: ts/mains/password_start.tsx (hunk 2 above)
Comment (Bilb) [reply ts/mains/password_start.tsx:12]: :+1:

File: _locales/en/messages.json (hunk 1 above)
Comment (Bilb) [reply _locales/en/messages.json:420]: amended

**Review by yougotwill — APPROVED** [review]
LGTM 🚀

**Pull request merged**